### ISSUE TITLE
Camera chime extender switch not working

### ISSUE THREAD

**Issue by cryptk** [natekspencer/hacs-vivint#121]
### The problem

When toggling the chime extender switch on a camera from home assistant, an error is returned rather than toggling the functionality. When toggling the chime extender switch from the Vivint app, the switch in home assistant does correctly reflect the change, it is only broken when toggling it from inside home assistant. 

### What version of Home Assistant Core are you running?

2023.11.2

### Anything in the logs that might be useful?

```txt
Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/commands.py:230
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 1:11:00 PM (1 occurrences)
Last logged: 1:11:00 PM

[548132673984] <AioRpcError of RPC that terminated with: status = StatusCode.UNAVAILABLE details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:206.79.220.190:443: Failed to connect to remote host: FD Shutdown" debug_error_string = "UNKNOWN:failed to connect to all addresses; last error: UNKNOWN: ipv4:206.79.220.190:443: Failed to connect to remote host: FD Shutdown {created_time:"2023-11-20T13:11:00.60188149-06:00", grpc_status:14}" >
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 230, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 2035, in async_call
    response_data = await coro
                    ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2072, in _execute_service
    return await target(service_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 235, in handle_service
    return await service.entity_service_call(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 876, in entity_service_call
    response_data = await _handle_entity_call(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 948, in _handle_entity_call
    result = await task
             ^^^^^^^^^^
  File "/config/custom_components/vivint/switch.py", line 143, in async_turn_on
    await self.entity_description.turn_on(self.device)
  File "/usr/local/lib/python3.11/site-packages/vivintpy/devices/camera.py", line 169, in set_as_doorbell_chime_extender
    await self.api.set_camera_as_doorbell_chime_extender(
  File "/usr/local/lib/python3.11/site-packages/vivintpy/vivintskyapi.py", line 191, in set_camera_as_doorbell_chime_extender
    response: beam_pb2.SetUseAsDoorbellChimeExtenderResponse = await stub.SetUseAsDoorbellChimeExtender(
                                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/grpc/aio/_call.py", line 318, in __await__
    raise _create_rpc_error(
grpc.aio._call.AioRpcError: <AioRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:206.79.220.190:443: Failed to connect to remote host: FD Shutdown"
	debug_error_string = "UNKNOWN:failed to connect to all addresses; last error: UNKNOWN: ipv4:206.79.220.190:443: Failed to connect to remote host: FD Shutdown {created_time:"2023-11-20T13:11:00.60188149-06:00", grpc_status:14}"
>
```

**Comment by natekspencer** [comment]
This should be resolved in 2023.11.0

**Comment by cryptk** [comment]
Just updated and tested, seems to be working great now! Thanks for the super fast fix!
